test/arg_stream/arg_stream_test.cpp

```diff
@@ -0,0 +1,211 @@
+/*
+    Copyright 2005-2013 Adobe Systems Incorporated
+    Distributed under the MIT License (see accompanying file LICENSE_1_0_0.txt
+    or a copy at http://stlab.adobe.com/licenses.html)
+*/
+
+/******************************************************************************/
+
+#include <adobe/config.hpp>
+
+// boost
+#define BOOST_TEST_MAIN
+#include <boost/test/unit_test.hpp>
+
+// asl
+#include <adobe/arg_stream.hpp>
```

Comment:
This should be the first file included to test that there are no hidden dependencies.